templates/geth/README.md

```diff
@@ -0,0 +1,37 @@
+# template: github.com/threefoldfoundation/tfchain-templates/geth/0.0.2
+
+## Description
+
+This template is responsible for deploying a geth.
```

Comment:
since `geth` stands for go-ethereum, "deploying a geth node" sounds better to me